models/engine/file_storage.py

```diff
@@ -68,3 +68,21 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """ Return a element if is found"""
+
+        if cls is not None:
```

Comment:
I think that and else condition for the CLS when is None, its necessary. to complete the requirements.